src/s1tools/sarhspredictor/test/test_predictions_from_ocn.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import os
+import s1tools
+from s1tools.sarhspredictor.predict_hs_from_wv_ocn_cartesian_xspec import (
+    get_hs_inference_from_cartesian_xspectra,
+)
+import numpy as np
+
+# expected = {"hs": 12.17, "hsstd": 1.473}
+expected = {"hs":11.87,"hsstd":1.546}
+
+def test_hs_predictions_from_ocn_file():
+    input_test_dataset = os.path.join(
+        os.path.dirname(s1tools.sarhspredictor.__file__),
+        "referencedata",
+        "S1A_WV_OCN__2SSV_20231003T122250_20231003T124111_050600_061886_3197.SAFE",
+        "measurement",
+        "s1a-wv2-ocn-vv-20231003t122403-20231003t122406-050600-061886-006.nc",
+    )
+    actual_ds_hs_predictions = get_hs_inference_from_cartesian_xspectra(input_test_dataset)
+    print("ds_hs_predictions", actual_ds_hs_predictions)
+    # expected_predictions = data['yhat']
+    # test Hs
+    assert np.allclose(actual_ds_hs_predictions["oswTotalHs"].values, expected["hs"], atol=1e-02)
+    # test Hs STD
+    assert np.allclose(actual_ds_hs_predictions["oswTotalHsStdev"], expected["hsstd"], atol=1e-02)
```

Comment:
Can you update the expected values after updating the config.yaml with the good models files

Reply:
Expected values were already updated. unit test are OK:
```bash
============================================================================================ test session starts =============================================================================================
platform linux -- Python 3.10.13, pytest-8.1.1, pluggy-1.4.0
rootdir: /home1/datahome/agrouaze/sources/git/sar_hs_predictor
configfile: pyproject.toml
collected 3 items                                                                                                                                                                                             

test_predictions.py ..                                                                                                                                                                                  [ 66%]
test_predictions_from_ocn.py .                                                                                                                                                                          [100%]

============================================================================================== warnings summary ===============================================================================================
src/s1tools/sarhspredictor/test/test_predictions_from_ocn.py::test_hs_predictions_from_ocn_file
  <frozen importlib._bootstrap>:241: RuntimeWarning: numpy.ndarray size changed, may indicate binary incompatibility. Expected 16 from C header, got 96 from PyObject

src/s1tools/sarhspredictor/test/test_predictions_from_ocn.py::test_hs_predictions_from_ocn_file
  /opt/conda-envs/mamba-l2awinddir-dev/lib/python3.10/site-packages/xarray/core/computation.py:822: RuntimeWarning: divide by zero encountered in log10
    result_data = func(*input_data)

src/s1tools/sarhspredictor/test/test_predictions_from_ocn.py::test_hs_predictions_from_ocn_file
  /opt/conda-envs/mamba-l2awinddir-dev/lib/python3.10/site-packages/xarray/core/computation.py:822: RuntimeWarning: invalid value encountered in sqrt
    result_data = func(*input_data)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================================================================================== 3 passed, 3 warnings in 7.11s ========================================================================================
```